examples/build-package/main.tf

```
@@ -63,6 +63,45 @@ module "package_dir_poetry" {
   artifacts_dir = "${path.root}/builds/package_dir_poetry/"
 }
 
+# Create zip-archive of a src directory where "poetry export" & "pip install --no-deps" will also be executed (using docker)
+module "package_src_poetry" {
+  source = "../../"
+
+  create_function = false
+
+  build_in_docker = true
+  runtime         = "python3.9"
+  docker_image    = "build-python3.9-poetry"
+  docker_file     = "${path.module}/../fixtures/python3.9-app-src-poetry/docker/Dockerfile"
+
+  source_path = [
+    "${path.module}/../fixtures/python3.9-app-src-poetry/src",
+    {
+      path           = "${path.module}/../fixtures/python3.9-app-src-poetry/pyproject.toml"
+      poetry_install = true
+    }
+  ]
+  artifacts_dir = "${path.root}/builds/package_src_poetry/"
+}
+
+# Create zip-archive of a src directory where "poetry export" & "pip install --no-deps" will also be executed (using docker)
+module "package_src_poetry2" {
+  source = "../../"
+
+  create_function = false
+
+  build_in_docker = true
+  runtime         = "python3.9"
+  docker_image    = "build-python3.9-poetry"
+  docker_file     = "${path.module}/../fixtures/python3.9-app-src-poetry/docker/Dockerfile"
+
+  source_path = [
+    "${path.module}/../fixtures/python3.9-app-src-poetry/src",
+    "${path.module}/../fixtures/python3.9-app-src-poetry/pyproject.toml"
+  ]
+  artifacts_dir = "${path.root}/builds/package_src_poetry/"
```

This should point to another directory:
```suggestion
  artifacts_dir = "${path.root}/builds/package_src_poetry2/"
```